indexer-app/src/cmr/indexer/data/concepts/track.clj

```diff
@@ -0,0 +1,25 @@
+(ns cmr.indexer.data.concepts.track
+  "Contains functions for converting track passes into elastic documents")
+
+(defn- tile->indexed-tiles
+  "Converts a pass track tile into tiles for indexing.
+  Track pass tile can be a Left, Right or Full tile, we index Full tile as both Left and
+  Right tiles, so we can find it by searching with either Left or Right tile.
+  e.g. 2F will be indexed as [2F 2L 2R]"
+  [tile]
+  (if-let [[_ tile-num] (re-matches #"(\d+)F" tile)]
```

Comment:
Didn't know you could destructure like that within an if-let.

Reply:
yeah, I have to try it to see it actually works.